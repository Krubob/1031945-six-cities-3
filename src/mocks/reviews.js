export default [
  {
    id: `4353434`,
    reviews: [
      {
        author: `Max`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 4,
        text: `A quiet cozy and picturesque that hides behind a a river
    by the unique lightness of Amsterdam. The building is
    green and from 18th century`,
        date: `April 23, 2019`
      },
      {
        author: `Mark`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 2,
        text: `A quiet cozy and picturesque that hides behind a a river
      by the unique lightness of Amsterdam. The building is
      green and from 18th century`,
        date: `May 18, 2018`
      },
      {
        author: `Mary`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 2.5,
        text: `A quiet cozy and picturesque that hides behind a a river
      by the unique lightness of Amsterdam. The building is
      green and from 18th century`,
        date: `December 11, 2019`
      },
      {
        author: `Daniel`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 2.8,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `March 17, 2020`
      }
    ]
  },
  {
    id: `3345674`,
    reviews: [
      {
        author: `John`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.4,
        text: `A quiet cozy and picturesque that hides behind a a river
      by the unique lightness of Amsterdam. The building is
      green and from 18th century`,
        date: `July 1, 2017`
      },
      {
        author: `Rob`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 1.4,
        text: `A quiet cozy and picturesque that hides behind a a river
      by the unique lightness of Amsterdam. The building is
      green and from 18th century`,
        date: `June 8, 2019`
      }
    ]
  },
  {
    id: `1225459`,
    reviews: [
      {
        author: `Alex`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 4.1,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `April 29, 2019`
      },
      {
        author: `Sam`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 4.3,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `October 12, 2019`
      },
      {
        author: `Vladimir`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.7,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `January 16, 2020`
      },
      {
        author: `Jack`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.9,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `July 9, 2016`
      }
    ]
  },
  {
    id: `4298509`,
    reviews: [
      {
        author: `Grigoriy`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 4.1,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `September 14, 2017`
      },
      {
        author: `Vasiliy`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 4.9,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `November 8, 2015`
      }
    ]
  },
  {
    id: `7275453`,
    reviews: [
      {
        author: `Mike`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.9,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `July 23, 2016`
      },
      {
        author: `Luise`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.3,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `December 15, 2017`
      },
      {
        author: `Gary`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.4,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `January 17, 2018`
      },
      {
        author: `Boris`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 2.7,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `November 3, 2018`
      }
    ]
  },
  {
    id: `2345675`,
    reviews: [
      {
        author: `Max`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 4,
        text: `A quiet cozy and picturesque that hides behind a a river
    by the unique lightness of Amsterdam. The building is
    green and from 18th century`,
        date: `April 23, 2019`
      },
      {
        author: `Mark`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 2,
        text: `A quiet cozy and picturesque that hides behind a a river
      by the unique lightness of Amsterdam. The building is
      green and from 18th century`,
        date: `May 18, 2018`
      },
      {
        author: `Mary`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 2.5,
        text: `A quiet cozy and picturesque that hides behind a a river
      by the unique lightness of Amsterdam. The building is
      green and from 18th century`,
        date: `December 11, 2019`
      },
      {
        author: `Daniel`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 2.8,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `March 17, 2020`
      }
    ]
  },
  {
    id: `6784673`,
    reviews: [
      {
        author: `John`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.4,
        text: `A quiet cozy and picturesque that hides behind a a river
      by the unique lightness of Amsterdam. The building is
      green and from 18th century`,
        date: `July 1, 2017`
      },
      {
        author: `Rob`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 1.4,
        text: `A quiet cozy and picturesque that hides behind a a river
      by the unique lightness of Amsterdam. The building is
      green and from 18th century`,
        date: `June 8, 2019`
      }
    ]
  },
  {
    id: `3422357`,
    reviews: [
      {
        author: `Alex`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 4.1,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `April 29, 2019`
      },
      {
        author: `Sam`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 4.3,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `October 12, 2019`
      },
      {
        author: `Vladimir`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.7,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `January 16, 2020`
      },
      {
        author: `Jack`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.9,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `July 9, 2016`
      }
    ]
  },
  {
    id: `8997812`,
    reviews: [
      {
        author: `Grigoriy`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 4.1,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `September 14, 2017`
      },
      {
        author: `Vasiliy`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 4.9,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `November 8, 2015`
      }
    ]
  },
  {
    id: `8874357`,
    reviews: [
      {
        author: `Mike`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.9,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `July 23, 2016`
      },
      {
        author: `Luise`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.3,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `December 15, 2017`
      },
      {
        author: `Gary`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.4,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `January 17, 2018`
      },
      {
        author: `Boris`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 2.7,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `November 3, 2018`
      }
    ]
  },
  {
    id: `2345675`,
    reviews: [
      {
        author: `Mike`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.9,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `July 23, 2016`
      }
    ]
  },
  {
    id: `6784673`,
    reviews: [
      {
        author: `Mike`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.9,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `July 23, 2016`
      },
      {
        author: `Boris`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 2.7,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `November 3, 2018`
      }
    ]
  },
  {
    id: `3422357`,
    reviews: [
      {
        author: `Mike`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.9,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `July 23, 2016`
      },
      {
        author: `Boris`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 2.7,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `November 3, 2018`
      }
    ]
  },
  {
    id: `8997812`,
    reviews: [
      {
        author: `Mike`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.9,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `July 23, 2016`
      },
      {
        author: `Boris`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 2.7,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `November 3, 2018`
      }
    ]
  },
  {
    id: `8874357`,
    reviews: [
      {
        author: `Mike`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.9,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `July 23, 2016`
      },
      {
        author: `Boris`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 2.7,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `November 3, 2018`
      }
    ]
  },
  {
    id: `9156899`,
    reviews: [
      {
        author: `Mike`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 3.9,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `July 23, 2016`
      },
      {
        author: `Boris`,
        avatar: {
          src: `img/avatar-max.jpg`,
          alt: `Reviews avatar`
        },
        rate: 2.7,
        text: `A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century`,
        date: `November 3, 2018`
      }
    ]
  }
];
