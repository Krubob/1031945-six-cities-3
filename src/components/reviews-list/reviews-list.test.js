import React from "react";
import renderer from "react-test-renderer";
import ReviewsList from "./reviews-list.jsx";
import reviews from "../../mocks/reviews";

it(`Render ReviewsList component`, () => {
  const tree = renderer
    .create(
        <ReviewsList review = {reviews[0]}/>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
