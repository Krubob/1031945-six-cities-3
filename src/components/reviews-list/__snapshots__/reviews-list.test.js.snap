// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ReviewsList component 1`] = `
<ul
  className="reviews__list"
>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height="54"
          src="img/avatar-max.jpg"
          width="54"
        />
      </div>
      <span
        className="reviews__user-name"
      >
        Max
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": 4,
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        A quiet cozy and picturesque that hides behind a a river
    by the unique lightness of Amsterdam. The building is
    green and from 18th century
      </p>
      <time
        className="reviews__time"
        dateTime="2019-04-24"
      >
        April 23, 2019
      </time>
    </div>
  </li>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height="54"
          src="img/avatar-max.jpg"
          width="54"
        />
      </div>
      <span
        className="reviews__user-name"
      >
        Mark
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": 2,
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        A quiet cozy and picturesque that hides behind a a river
      by the unique lightness of Amsterdam. The building is
      green and from 18th century
      </p>
      <time
        className="reviews__time"
        dateTime="2019-04-24"
      >
        May 18, 2018
      </time>
    </div>
  </li>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height="54"
          src="img/avatar-max.jpg"
          width="54"
        />
      </div>
      <span
        className="reviews__user-name"
      >
        Mary
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": 2.5,
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        A quiet cozy and picturesque that hides behind a a river
      by the unique lightness of Amsterdam. The building is
      green and from 18th century
      </p>
      <time
        className="reviews__time"
        dateTime="2019-04-24"
      >
        December 11, 2019
      </time>
    </div>
  </li>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height="54"
          src="img/avatar-max.jpg"
          width="54"
        />
      </div>
      <span
        className="reviews__user-name"
      >
        Daniel
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": 2.8,
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        A quiet cozy and picturesque that hides behind a a river
        by the unique lightness of Amsterdam. The building is
        green and from 18th century
      </p>
      <time
        className="reviews__time"
        dateTime="2019-04-24"
      >
        March 17, 2020
      </time>
    </div>
  </li>
</ul>
`;
